tools/test.py

```diff
@@ -81,8 +81,6 @@ def parse_args():
         'in  data config.')
     parser.add_argument(
         '--uri',
-        action='store_true',
```

Comment:
Why remove these two lines?

Reply:
I think uri is str, and there shoudn't have action='store_true'

Reply:
Will it affect snpe test?